.github/workflows/dev-deploy-web.yml

```
@@ -40,6 +40,8 @@ jobs:
       netlify_alias: dev
       netlify_gh_env: development
       netlify_site_id: 45396446-dc86-4ad6-81e4-86d3eb78d06f
+      clerk_publishable_key: {{ secrets.CLERK_PUBLISHABLE_KEY }}
```

Nit-picky one, since this is a hacky naming, let's make it clear.
```suggestion
      clerk_publishable_key: {{ secrets.CLERK_PUBLISHABLE_KEY_STAGING }}
```

---

we will be changing this anyway, let's leave it for now